_data/versions.yml

```diff
@@ -2030,3 +2030,9 @@ master:
       url: ""
      flannel:
       version: v0.9.1
+     calico/dikastes:
+      version: 20180328162858
```

Comment:
Interesting, I guess we won't follow the typical versioning scheme with the new components? I am guessing that this change does not break the build, but it can be a bit touchy, so might be worth verifying that if you haven't already.

Reply:
We will eventually have released versions of this component, but for now these are just tagged CI builds that are "known good"